### ISSUE TITLE
[Bug Report][3.4.0] Accessibility: v-overlay tooltip needs aria-label

### ISSUE THREAD

**Issue by kmmerc18** [vuetifyjs/vuetify#18682]
### Environment
**Vuetify Version:** 3.4.0
**Vue Version:** 3.3.7
**Browsers:** Chrome 119.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Go to the reproduction link, click the "Show Overlay" button to show the overlay, then use the Chrome axeDevTools extension to scan the page for errors. The issue in question is this:
![Screenshot 2023-11-14 at 7 24 20 AM](https://github.com/vuetifyjs/vuetify/assets/60188830/32a42e16-826e-4df0-bf45-578aefc1910c)

### Expected Behavior
All elements with a role of 'tooltip' should have an aria-label

### Actual Behavior
There is no aria-label on the div for v-overlays with role of 'tooltip' 

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpVUMtugzAQ/JWtDxFI4aGmUiUEqP2CHtpb6cExG4FqMLIXkiri32vexBfvzM7OPr7vzGgRvDeN37XIIhYTVo3khGlWA8R52YGQ3JgkY4Q38gTWhDpjY9oKOu9MNQgllbYS1FrZJLwJWYpfS6gOteR/kMDTHNpS+CzUFT7mVByMHtZwtVyquC65J/kZ5WZl3TuvUvkjlw4uM5omD+zoNoqD3UIWGqHLhkYJ3hqlCXK88FYS3Kf2OScegeNCkoIzcwCzdQQXLg0eJ7p3bTCFV06iiBaPVe90XLobC2AxHA4bBjBIX2WFqiVnarpTD4+K0vjbGcf+e0V/hOcwDN2F65fhxr8fLrDuzPojO/kn/5UN/4sfsp9/SzSdew==)

**Comment by KaelWD** [comment]
There is no tooltip in your example, and tooltips already set `aria-describedby` if they have an activator.

**Comment by kmmerc18** [comment]
@KaelWD I've re-checked, and still see this issue. The v-overlays are divs with a role of tooltip, but no aria label. The example includes a v-overlay with this issue
